.isort.cfg

```
@@ -1,2 +1,2 @@
 [settings]
-known_third_party = altair,autoload_data,cachetools,data_wranglers,datapane,db,deco,docbao_api,dotenv,fastapi,flask,flask_sqlalchemy,jsonpickle,jwt,lib,more_itertools,nltk,numpy,pandas,pika,psycopg2,pydantic,pydub,pytest,qcore,schedule,scipy,sklearn,sqlalchemy,starlette,topdup_app,tqdm,vncorenlp
+known_third_party = altair,autoload_data,bs4,cachetools,data_wranglers,datapane,db,deco,docbao_api,dotenv,fastapi,flask,flask_sqlalchemy,jsonpickle,jwt,lib,libs,loguru,lxml,more_itertools,nltk,numpy,pandas,pika,psycopg2,pydantic,pydub,pytest,pytz,qcore,requests,schedule,scipy,selenium,sklearn,sqlalchemy,starlette,topdup_app,tqdm,vncorenlp,yaml
```

Sao lại có diff này ở PR này nhỉ?
```suggestion
known_third_party = altair,autoload_data,cachetools,data_wranglers,datapane,db,deco,docbao_api,dotenv,fastapi,flask,flask_sqlalchemy,jsonpickle,jwt,lib,libs,loguru,lxml,more_itertools,nltk,numpy,pandas,pika,psycopg2,pydantic,pydub,pytest,pytz,qcore,requests,schedule,scipy,selenium,sklearn,sqlalchemy,starlette,topdup_app,tqdm,vncorenlp,yaml
```

---

Chắc do em lâu rồi không cập nhật từ nhánh staging chăng @.@